### PR TITLE
Add initial amp-kaltura-player sample

### PR DESCRIPTION
**Pull request by jmadler**
#252

### CODE REVIEW

File: src/20_Components/amp-kaltura-player.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    Embed videos hosted on Kaltura via the `data-uiconf`, `data-partner`, and `data-entryid` attributes.  We recommend using the responsive layout for videos.
+    The `data-param-*` attributes will be added as key/value pairs to the query string in the video player's embed iframe, and can be used to pass ad parameters to Kaltura's video player, for example.
+  -->
+    <amp-kaltura-player
+      data-uiconf="33502051"
+      data-partner="1281471"
+      data-entryid="1_3ts1ms9c"
+      data-param-streamerType="auto"
+      layout="responsive"
+      width="480" height="720"></amp-vimeo>
```
Comment (kul3r4) [inline src/20_Components/amp-kaltura-player.html]: </amp-vimeo> -> </amp-kaltura-player>

File: src/20_Components/amp-kaltura-player.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  #### Introduction
+
+  Embed Kaltura videos in your AMP HTML files.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/components/amp-vimeo/">
```
Comment (kul3r4) [inline src/20_Components/amp-kaltura-player.html]: amp-vimeo -> amp-kaltura

**Comment by jmadler** [comment]
PTAL

**Comment by kul3r4** [comment]
LGTM, could you squash the commits?

**Pull request merged**